### PR TITLE
UI to add new course

### LINKED ISSUE #439: Web interface for creating a new course
A `New Course` button should appear here: `https://ok-server.appspot.com/manage#/course/`

### PR DESCRIPTION
**Pull request by alvinwan**
fixes #439 

<strike>It's not perfect; there's no redirect after creating the course, and the view then doesn't auto-refresh, but the creation menu works now.</strike>
Changes:
- create menu added
- after creation, redirected back to course list

Caveats:
- *course list does not refresh*

v2:
site-new-course

v1:
site-new-course

### CODE REVIEW

File: server/static/js/admin/controllers.js [hunk 1]
```diff
@@ -298,14 +298,24 @@ app.controller("CourseDetailCtrl", ["$scope", "$stateParams", "Course",
...
-      course.$save();
+    $scope.createCourse = function() {
+      Course.create({
+        'display_name': $scope.course.name,
+        'institution': $scope.course.institution,
+        'offering': $scope.course.offering,
+        'active': true
+      },
+       function (response) {
+         $window.swal("Course Created!",'','success');
```
Comment (Sumukh) [inline server/static/js/admin/controllers.js]: You can add the redirect back to the course page here.

Comment (alvinwan) [inline server/static/js/admin/controllers.js]: Thanks, added.

**Comment by alvinwan** [comment]
@Sumukh awaiting your review

File: server/static/js/admin/controllers.js [hunk 2]
```diff
@@ -298,14 +298,25 @@ app.controller("CourseDetailCtrl", ["$scope", "$stateParams", "Course",
   }
   ]);
 
-app.controller("CourseNewCtrl", ["$scope", "Course",
-  function ($scope, Course) {
+app.controller("CourseNewCtrl", ["$scope", "$state", "$window", "Course",
+  function ($scope, $state, $window, Course) {
     $scope.course = {};
     $scope.test = {'test':3};
```
Comment (Sumukh) [inline server/static/js/admin/controllers.js]: I'm going to remove this.

Comment (alvinwan) [inline server/static/js/admin/controllers.js]: The two params? We need $state for transitionTo and $window for the alert, though.

**Comment by Sumukh** [comment]
Cool - I just fixed the reload/data issue and changed the table/button layout. Merging now.

**Comment by robhuang** [comment]
Where is this magic button? I only see this:
screen shot 2015-06-19 at 18 46 52

**Comment by alvinwan** [comment]
It migrated. I didn't know how to prettify it, so Sumukh helped. It now lives in the table header:

screen shot 2015-06-19 at 7 01 50 pm

**Comment by robhuang** [comment]
I'm not sure what you mean by pulling, and refreshing didn't change it. I'm on https://ok-server.appspot.com/manage#/course/

**Comment by Sumukh** [comment]
@alvinwan : I think Robert is referring to the deployed version on app engine. I don't believe anything has been deployed.

**Comment by alvinwan** [comment]
@Sumukh Oh, I see.

**Pull request merged**